README.md

```diff
@@ -51,6 +56,20 @@ After starting your container, you can get the randomly generated SuperUser pass
 
 **--- OR ---**
 
+Provide a SuperUser password using the `SUPERUSER_PASSWORD` environment variable:
+
+```
+    docker run -d \
+      -e SUPERUSER_PASSWORD=mysupersecurepassword
+      -p 64738:64738 \
+      -p 64738:64738/udp \
+      -v mumble-data:/etc/mumble \
+      --name mumble-server \
+      phlak/mumble
+```
+
+**--- OR ---**
+
```

Comment:
Let's move this to the `Optional 'docker run' arguments` section above. Something like this:

    `-e SUPERUSER_PASSWORD=password` - Set the superuser password during container initialization...

Then maybe just add a note here that references the optional run argument.

Reply:
done